integration/src/main/java/com/aws/greengrass/mqtt/moquette/BrokerKeyStore.java

```diff
@@ -75,10 +76,12 @@ public void initialize() throws KeyStoreException {
      * @throws KeyStoreException If unable to set key entry.
      */
     public void updateServerCertificate(CertificateUpdateEvent certificateUpdate) throws KeyStoreException {
-        // TODO: Support certificate chains
-        Certificate[] certChain = {certificateUpdate.getCertificate()};
-        jks.setKeyEntry(BROKER_KEY_ALIAS, certificateUpdate.getKeyPair().getPrivate(), jksPassword.toCharArray(),
-            certChain);
+        X509Certificate[] fullChain = Stream.concat(Stream.of(certificateUpdate.getCertificate()),
```

Comment:
Don't know if it would be very useful but it would be nice to have a unit tests that passes multiple ca's and a certificate with the certificate update event and we ensure the are all on the key store

Reply:
Yep, can work on a test, but wanted to get this merged to unblock testing